test/node.js

```diff
@@ -438,7 +447,7 @@ node.initApplication = function (cb, initScope) {
 			},
 			network: function (cb) {
 				// Init with empty function
-				cb(null, {io: {sockets: {emit: function () {}}}});
+				cb(null, {io: {sockets: {emit: function () {}}}, app: require('express')()});
```

Comment:
Are you sure that we need it and it will not conflict with currently running tests?

Reply:
We need an express app to mount the swagger middleware. Without it we can't test the swagger files properly. Yes all tests are passing.